test/net.test.js

```diff
@@ -12,39 +12,37 @@ describe("CALCULATE NET SALARY:", function () {
       test('Gross is equal 12.290503m', () => {
 
 
-        let netSalary = net(12290503, 0);
-        let roundedSalary = Math.round(netSalary);
+        let netSalary = net(12_290_503, 0, 1, new Date("2022-04-01"));
 
-        assert.equal(roundedSalary, 11000000);
+        assert.equal(netSalary, 11_000_000.17575);
       });
 
       test('Gross is lt 12.290500m', function () {
         const netSalary = net(2000000, 0);
 
-        assert.equal(netSalary.toNumber(), 1790000);
+        assert.equal(netSalary, 1790000);
       });
 
       test('Gross is almost 0', function () {
         const netSalary = net(3, 0);
 
-        assert.equal(netSalary.toNumber(), 2.685);
+        assert.equal(netSalary, 2.685);
       });
 
       test('Gross is eq 0', function () {
         const netSalary = net(0, 0);
 
-        assert.equal(netSalary.toNumber(), 0);
+        assert.equal(netSalary, 0);
       });
     })
 
     describe("HAVING DEPENDENTS", function () {
 
-      test('Gross is equal 12.290503m ', () => {
+      test('Gross is equal 12.290503m, dep 1, region 1 ', () => {
```

Comment:
`() =>` this sync tax called Arrow Function. Please replace it by `function()` since you've not learned the arrow function yet.